ui/app/components/pages/create-account/index.js

```diff
@@ -75,7 +75,7 @@ const mapDispatchToProps = dispatch => ({
     dispatch(actions.showModal({ name: 'EXPORT_PRIVATE_KEY' }))
   },
   hideModal: () => dispatch(actions.hideModal()),
-  saveAccountLabel: (address, label) => dispatch(actions.saveAccountLabel(address, label)),
+  setAccountLabel: (address, label) => dispatch(actions.setAccountLabel(address, label)),
```

Comment:
You probably could have saved yourself a lot of work by keeping the name `saveAccountLabel`, but this name is better, I just hope you didn't miss any instances!

Reply:
I think I got all the instances 😄 and agreed, I think this name is clearer